lxqt-admin-time/translations/lxqt-admin-time_pt.ts

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.0" language="pt">
+<context>
+    <name>TimeAdminDialog</name>
+    <message>
+        <location filename="../timeadmindialog.ui" line="14"/>
+        <source>Time and Date Settings</source>
+        <translation>Definições de data e hora</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="23"/>
+        <source>Set current date and time</source>
+        <translation>Definir data e hora atual</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="30"/>
+        <source>Time zone:</source>
+        <translation>Fuso horário:</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="40"/>
+        <source>Time:</source>
+        <translation>Hora:</translation>
+    </message>
+    <message>
+        <location filename="../timeadmindialog.ui" line="47"/>
+        <source>hh:mm:ss AP</source>
```

Comment:
@smarquespt  lxqt-admin-user/translations/lxqt-admin-user_pt.ts wasn't translated using the latest .ts